plugins/http_plugin/http_plugin.cpp

```diff
@@ -87,11 +87,11 @@ class http_plugin_impl : public std::enable_shared_from_this<http_plugin_impl> {
 
                // post to the app thread taking shared ownership of next (via std::shared_ptr),
                // sole ownership of the tracked body and the passed in parameters
-               app().post( priority, [next_ptr, conn=std::move(conn), r=std::move(r), b = std::move(b), wrapped_then=std::move(wrapped_then)]() mutable {
+               app().post( priority, [next_ptr=std::move(next_ptr), conn=std::move(conn), r=std::move(r), b = std::move(b), wrapped_then=std::move(wrapped_then)]() mutable {
```

Comment:
```suggestion
               app().post( priority, [next_ptr=next_ptr, conn=std::move(conn), r=std::move(r), b = std::move(b), wrapped_then=std::move(wrapped_then)]() mutable {
```
interesting catch from clang-tidy, but I think it is minor:
`Std::move of the const variable 'next_ptr' has no effect; remove std::move() or make the variable non-const`

Reply:
Well actually I undid my change at line 90, because I have a weird error when `next_ptr` is actually moved.